.github/workflows/tests.yml

```diff
@@ -0,0 +1,69 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+  pull-request:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+
+jobs:
+  build_wheels:
+    name: ${{ matrix.platform }}
+    runs-on: ${{ matrix.image }}
```

Comment:
@jonathf did you change that because it didn't hit the cache? just had the exact same problem for other bindings, 2 secs later I see this PR;) you just try or is that a recommended solution?

Reply:
The adding of the file is to have a lighter build that runs on pull-requests and main.
The line in question of splitting the os into image and platform is just estetics.